source/_integrations/shelly.markdown

```diff
@@ -51,7 +51,7 @@ There are three generations of devices and all generations are supported by this
 
 Generation 1 devices use the `CoIoT` protocol to communicate with the integration. `CoIoT` must be enabled in the device settings. Navigate to the local IP address of your Shelly device, **Internet & Security** > **ADVANCED - DEVELOPER SETTINGS** and check the box **Enable CoIoT**.
 
-We recommend using `unicast` for communication. To enable this, enter the local IP address of the Home Assistant server and port `5683` into the **CoIoT peer** field and push **SAVE** button. **This is mandatory for battery operated devices**. After changing the **CoIoT peer**, the Shelly device needs to be manually restarted.
+We recommend using `unicast` for communication. To enable this, enter the local IP address of the Home Assistant server and port `5683` into the **CoIoT peer** field and push **SAVE** button. **This is mandatory for sleeping devices** (usually battery operated). After changing the **CoIoT peer**, the Shelly device needs to be manually restarted.
```

Comment:
To be honest, I doubt that users will know what "sleeping device" means. I think it will be better to add information that unicast is obligatory also for battery-powered devices when they are powered by USB.

Reply:
Re-worded